support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -204,6 +204,12 @@ function userInTest(
 	}
 
 	if (referrerControlled) {
+		const isSfdV2Test = acquisitionDataTest?.name.startsWith('SFD_V2');
+
+		if (isSfdV2Test) {
+			return isSfdV2Test;
+		}
+
 		return acquisitionDataTest && acquisitionDataTest.name === testId;
```

Comment:
I'll give this a go locally but have a feeling it may break some tests. Initially refactored this to return a boolean and had to change it back to make them happy

Reply:
Might be worth fixing the tests? We should really only be returning boolean here.

Reply:
This works! There must've been something off about the series of if statements I'd tried